examples/plotting/file/box_annotation.py

```diff
@@ -1,23 +1,30 @@
+''' A timeseries plot of glucose data readings demonstrates adding box
+annotations.
+
+.. bokeh-example-metadata::
+    :sampledata: :ref:`sampledata_glucose`
+    :apis: :func:`~bokeh.plotting.Figure.line`, :func:`~bokeh.plotting.Figure.scatter`, :class:`~bokeh.models.annotations.BoxAnnotation`
```

Comment:
This looks very opaque to me. Is there really no way to make Sphinx to know what a unique, fully qualified name refers to?

Reply:
Sphinx does not auto-reference in this context. (AFAIK it might only be params in dosctrings that get that treatment)

Reply:
So, we can have the directive put `:class` in front of everything unconditionally. The difference is e.g. `line` shows up instead of `line()`. I think that is fine (maybe even better) I will make that change.